game/scripts/npc/items/custom/item_sonic.txt

```diff
@@ -0,0 +1,92 @@
+"DOTAItems"
+{
+  //=================================================================================================================
+  // Recipe: Sonic
+  //=================================================================================================================
+  "item_recipe_sonic"
+  {
+    "ID"                  "8501"
+    "BaseClass"           "item_datadriven"
+    "Model"               "models/props_gameplay/recipe.mdl"
+    "AbilityTextureName"  "custom/recipe/recipe_3"
+    "ItemCost"            "3500"
+    "ItemShopTags"        ""
+    "ItemRecipe"          "1"
+    "ItemResult"          "item_sonic"
+    "ItemRequirements"
+    {
+      "01"                "item_ancient_janggo_2;item_upgrade_core_2"
+      "02"                "item_ancient_janggo_2;item_upgrade_core_3"
+      "03"                "item_ancient_janggo_2;item_upgrade_core_4"
+
+    }
+  }
+
+  //=================================================================================================================
+  // Sonic
+  //=================================================================================================================
+  "item_sonic"
+  {
+    "ID"                  "8502"
+    "BaseClass"           "item_lua"
+    "AbilityBehavior"			"DOTA_ABILITY_BEHAVIOR_NO_TARGET | DOTA_ABILITY_BEHAVIOR_IGNORE_CHANNEL"
+    "AbilityTextureName"  "custom/sonic"
+    "ScriptFile"          "items/sonic.lua"
+
+    "AbilityCooldown"     "8.0"
```

Comment:
Needs a shared cooldown key to make all the levels share a cooldown, though I guess it doesn't matter much in this case. I don't remember what the key is called exactly, or where it's usually put. Refer to other item KVs.